### DISCUSSION TITLE
Incorrect lotus-miner scheduling

### DISCUSSION THREAD

**Discussion by dodohack** [filecoin-project/lotus#7952] [Storage Provider] [answered]
My lotus-worker has lots of free memory and swap to accept jobs, but lotus-miner refuses to do so, you can see from the log that the scheduler says the worker has not enough memory. 

The worker is only running a PC2 jobs actually.

lotus version v1.2.1.

![Screen Shot 2020-11-25 at 9 37 42 AM](https://user-images.githubusercontent.com/19568348/100209990-63753d80-2f02-11eb-8301-160393409cd9.png)

![Screen Shot 2020-11-25 at 9 44 07 AM](https://user-images.githubusercontent.com/19568348/100210340-cbc41f00-2f02-11eb-9925-f7208111dd85.png)

**Comment by Reiers** [comment]
Hi @dodohack 

Thanks for the report. The issue you are experiencing lies elsewhere.

Moving this issue to Lotus Discussions for help and troubleshooting.

Closing Lotus issue ticket.

**Comment by SgtCoin** [comment, marked as answer]
@dodohack 

Hi,

The latest version of Lotus includes various enhancements to the scheduler and also supports cgroups on Linux. I suggest you upgrade tot he latest version and retest as needed. Since this case is a little old, I am going to archive it for now, but please let me know if you have any questions. We will be happy to help out.
